omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -166,7 +166,7 @@
                 link "#{install_dir}/embedded/bin/python3", "#{install_dir}/embedded/bin/python"
 
                 delete "#{install_dir}/embedded/bin/2to3"
-                link "#{install_dir}/embedded/bin/2to3-3.7", "#{install_dir}/embedded/bin/2to3"
```

Comment:
This seems broken. Do you know if this link is used? We might want this for 7.22.

Reply:
This has probably been broken since #4464 (which is when we moved from Python 3.7 to Python 3.8). Those changes were shipped on 7.18 so if something important is failing it is not a recent failure.

Reply:
👍 True, if it's broken it's been broken for a while 🙄